water.md

```diff
@@ -20,59 +20,172 @@ Water demand representations have been constructed for six major sectors: agricu
+36|Scheldt|86|Africa-North West Coast|136|Isthmus of Tehuantepec|186|East Brazil-South Atlantic Coast
+37|Russia-South East Coast|87|Nile|137|Irrawaddy|187|Parnaiba
+38|Ural|88|Persian Gulf Coast|138|Sittang|188|Zambezi
+39|Dnieper|89|Indus|139	|Peninsula Malaysia|189|Australia-East Coast
+40|St Lawrence|90|Farahrud|140|Krishna|190|Africa-Indian Ocean Coast
+41|France-West Coast|91|Baja California|141|Andaman-Nicobar Islands|191|Australia-West Coast
+42|Gobi Interior|92|Plateau of Tibet Interior|142|Africa-West Coast|192|Sao Francisco 
+43|Amur|93|Red Sea-East Coast|143|Caribbean Coast|193|Australia-Interior 
+44|Loire|94|Arabian Peninsula|144|Africa-North Interior|194|Orange
+45|Caspian Sea Coast|95|Dead Sea|145|India East Coast|195|Uruguay-Brazil-South Atlantic Coast
+46|Seine|96|Mexico-Northwest Coast|146|Chao Phraya|196|Namibia-Coast
+47|Black Sea-North Coast|97|Helmand|147|Pennar|197|Africa-South Interior
+48|Yenisey|98|Sinai Peninsula|148|Gulf of Thailand Coast|198|South Africa-South Coast
+49|Dniester|99|Eastern Jordan-Syria|149|Niger|199|Limpopo
+50|Italy-East Coast|100	|Africa-Red Sea-Gulf of Aden Coast|150|Micronesia|200|La Puna Region
+
+
+
+### Water Sources
+
+Three distinct sources of fresh water are modeled, renewable water (surface and ground), non-renewable groundwater, and desalinated water. Renewable water is water that is replenished naturally by surface runoff and subsurface infiltration and release. Non-renewable groundwater is fossil groundwater resources or groundwater where abstraction exceeds recharge. Renewable water and non-renewable ground water are separately modeled for each basin. Desalinated water is available as an additional source of freshwater within each basin and for alternative end-use demands for water.
+
+Renewable water supplies are based on runoff estimates from Xanthos, a detailed global hydrology model ([Liu et al. 2017](water.html#liu2017); [Vernon et al. 2019](water.html#vernon2019)). Xanthos accounts for surface and subsurface processes to compute runoff at 0.5° grid resolution. Global climate datasets are utilized in conjunction with Xanthos to determine historical annual average runoff aggregated for each basin ([Turner et al. 2019b](water.html#turner2019b); [Liu et al. 2017](water.html#liu2017); [Vernon et al. 2019](water.html#vernon2019)). Of the total basin runoff, water available or accessible for human use takes into consideration requirements for ecosystem services, inaccessibility due to rapid flow and remote locations, and capacity of reservoir storage. Accessible fractions of total runoff vary across basins. Renewable water volumes up to the accessible fraction is available at nominal or low cost. Additional renewable water beyond the accessible fraction is available at a sharply higher cost to ensure availability of water for ecosystem services and to reflect capital investments necessary for reservoir expansion.
```

Comment:
Should this be the Liu 2019 reference?

Reply:
Oops, it's the wrong Liu. It should have been the Yaling Liu paper from 2018. I will send you a revised water.md file with corrected reference. Thx.